dist/index.js

```diff
@@ -47145,10 +47145,22 @@ const VALIDATORS = {
   user: userEntityV1alpha1Validator,
 }
 
+function modifyPlaceholders(obj) {
```

Comment:
these files withi `dist` are generated for GitHub actions (GHA also generates them automatically) and the recommended approach is to include these into the repo